jupyter_sigplot/sigplot.py

```diff
@@ -61,12 +61,11 @@ def __init__(self, *args, **kwargs):
         self.arrays = []
         # Where to look for data, and where to cache/symlink remote resources
         # that the server or client cannot access directly.
-        #
-        # TODO (sat 2018-11-16): This actually needs to be relative to the
-        # effective root of the notebook server, not just this kernel. See
-        # Github issue #17. I expect that we'll be able to just set
-        # self.data_dir to the notebook server's cwd/root and be good to go.
-        self.data_dir = kwargs.pop('data_dir', '')
+        self.notebook_dir = os.getcwd()
+        self.data_dir = kwargs.pop('data_dir', self.notebook_dir)
+        if not os.path.isabs(self.data_dir):
+            # Need to store absolute path of data directory in case user os.chdir
+            self.data_dir = os.getcwd() + "/" + self.data_dir
```

Comment:
Suggest using `unravel_path` here, or at least `os.path.join`, instead of explicit string manipulation